utils/try_require.js

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// Performs a safe require to the given path
```

Comment:
I know it's not Go, but I wonder if we might start writing comments in the Godoc style:
```suggestion
// tryRequire performs a safe require ...
```

Reply:
✅